src/test/spec/runner/test_event.rs

```diff
@@ -48,46 +36,16 @@ impl Matchable for TestEvent {
                 }
                 actual_command.matches(expected_command)
             }
-            (
```

Comment:
The v2 runner only actually asserts on command started events, so I removed the others to simplify things.